presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,76 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries
+  that enable spilling and that are moved to the reserved memory pool.
+* Fix a bug that can cause the planner to go into an infinite loop. This bug was affecting queries
+  with multiple predicates that couldn't be pushed down to the connectors.
+* Fix a bug that may cause queries with ``row_number()`` to fail or return incorrect results.
+  If the query had an equality predicate against ``-1`` or ``0`` the bug would cause the query to fail.
+  If the query had a less-than-or-equal-to predicate against a negative value the bug would cause
+  the query to produce incorrect results.
+* Fix query analysis failure when lambda is used in ``UNNEST`` or ``VALUES``.
+* Add the peak of total non-revocable memory reservation (user + system memory) to query stats.
+* Add an optimizer rule that rewrites table scans in queries against partitioned tables to an empty relation
```

Comment:
Improve performance of queries that filter all data from a table up-front (e.g., due to partition pruning).